Website/components/collectionFeaturedBanner.tsx

```diff
@@ -106,7 +108,7 @@ const CollectionFeaturedBanner = ({ featuredLaunch, isHomePage }: CollectionFeat
                                                 featuredLaunch && featuredLaunch.keys && featuredLaunch.keys[CollectionKeys.CollectionMint]
                                                     ? featuredLaunch.keys[CollectionKeys.CollectionMint].toString()
                                                     : ""
-                                            }`}
+                                            }${PROD ? "" : `?cluster=devnet`}`}
```

Comment:
just make a utils function that does this and use it rather than making this prod check everywhere ,it will need to be both for tokens and for collections